src/testing-control-panel/index.js

```diff
@@ -26,17 +26,21 @@ export default class TestingControlPanel extends PureComponent {
             />
           )}
         </UserWebBrowserConsumer>
-        <WebBrowserConsumer>
+        <RockAuthedWebBrowser>
           {(openUrl) => (
             <TouchableCell
               handlePress={() =>
-                openUrl('https://rock.newspring.cc', {}, { useRockToken: true })
+                openUrl(
+                  'https://apollosrock.newspring.cc',
```

Comment:
We probably still want this to be `rock.newspring.cc`.

Reply:
haha yes

Reply:
Thanks for fixing this!